web/client/components/geostory/builder/Settings.jsx

```diff
@@ -125,6 +125,7 @@ export default ({
                 <div><Message msgId="geostory.builder.settings.fontSize"/></div>
                 <div>
                     <Select
+                        disabled={!settings.isTitleEnabled}
                         value={settings.storyTitleFontSize}
                         onChange={event => onUpdateSettings("storyTitleFontSize", event.value)}
```

Comment:
If you click the x inside the select input an error show up in console

![image](https://user-images.githubusercontent.com/19175505/93858343-4763de80-fcbc-11ea-9527-56d8cadf42ac.png)

I think it's enough to set the `clearable` prop of react-select to false or check when event is undefined